src/DynamoCoreWpf/UI/Themes/Modern/OutPorts.xaml

```diff
@@ -208,23 +213,45 @@
                     Background="{StaticResource NodeTransientOverlayColor}"
                     IsHitTestVisible="True"
                     SnapsToDevicePixels="True"
-                    Visibility="{Binding NodeAutoCompleteMarkerVisible, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}">
-                <Label
-                    Name="NodeAutoCompleteMarkerLabel"
-                    FontSize="12"
-                    Width="25"
-                    Height="25"
-                    Margin="-3,-3,0,0"
-                >✨</Label>
-
+                    Visibility="{Binding NodeAutoCompleteMarkerEnabled, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}">
+                <Image
```

Comment:
I'm a little bit worried about increased memory consumption when adding an image for every port. But we can get away with ✨as just a character. Maybe use it's Unicode representation &#x2728 ?!